cylc/rose/utilities.py

```diff
@@ -325,6 +330,103 @@ def merge_rose_cylc_suite_install_conf(old, new):
     return old
 
 
+def invalid_defines_check(defines: List) -> None:
+    """Check for defines which do not contain an = and therefore cannot be
+    valid
+
+    Examples:
+
+        # A single invalid define:
+        >>> import pytest
+        >>> with pytest.raises(InvalidDefineError, match=r'\\* foo'):
+        ...    invalid_defines_check(['foo'])
+
+        # Two invalid defines and one valid one:
+        >>> with pytest.raises(
+        ...     InvalidDefineError, match=r'\\* foo.*\\n.* \\* bar'
+        ... ):
+        ...     invalid_defines_check(['foo', 'bar52', 'baz=442'])
+
+        # No invalid defines
+        >>> invalid_defines_check(['foo=12'])
+    """
+    invalid_defines = []
+    for define in defines:
+        if parse_cli_defines(define) == 'Invalid':
+            invalid_defines.append(define)
+    if invalid_defines:
+        msg = 'Invalid Suite Defines (should contain an =)'
```

Comment:
Are we still using "Suite" as far as Rose is concerned?

Reply:
Yes - It's a suite of configurations for a workflow.